lib/testing/src/lib/process-services-cloud/pages/edit-task-filter-cloud-component.page.ts

```diff
@@ -38,14 +38,19 @@ export class EditTaskFilterCloudComponentPage {
     saveButton: ElementFinder = element(by.css('[data-automation-id="adf-filter-action-save"]'));
     saveAsButton: ElementFinder = element(by.css('[data-automation-id="adf-filter-action-saveAs"]'));
     deleteButton: ElementFinder = element(by.css('[data-automation-id="adf-filter-action-delete"]'));
+    filter: ElementFinder = element(by.css(`adf-cloud-edit-task-filter mat-expansion-panel-header`));
 
     editTaskFilterDialogPage = new EditTaskFilterDialogPage();
 
     editTaskFilterDialog(): EditTaskFilterDialogPage {
         return this.editTaskFilterDialogPage;
     }
 
-    async clickCustomiseFilterHeader(): Promise<void> {
+    async isFilterDisplayed(): Promise<Boolean> {
```

Comment:
a filter is displayed when it is "present" ? shouldn't it be visible?

Reply:
yes